src/librbd/cache/ReplicatedWriteLog.cc

```diff
@@ -1240,7 +1240,7 @@ bool ReplicatedWriteLog<I>::alloc_resources(C_BlockIORequestT *req) {
   uint64_t bytes_dirtied = 0;
   uint64_t num_lanes = 0;
   uint64_t num_unpublished_reserves = 0;
-  uint64_t num_log_entries = 1;
+  uint64_t num_log_entries = 0;
```

Comment:
Nit: disregard some of the previous comments since this commit overrides them. It would be preferable if you didn't introduce a change in commit (a) and then back it out in commit (b), though. Can you instead merge these changes into their associated commits from earlier in this PR?

Reply:
Okay. I'll merge these changes into their associated commits.

Reply:
@dillaman Done